docs/features/lists.md

```diff
@@ -11,15 +11,45 @@ keywords:
   - "Contributor Lists Management"
 ---
 
-The Lists feature enables you to categorize, monitor, and analyze different groups of contributors within open source projects. With the Lists feature, gain granular insights into each contributor's activity and contributions.
+The OpenSauced Lists feature enables you to categorize, monitor, and analyze different groups of contributors within open source projects. With the Lists feature, gain granular insights into each contributor's activity and contributions.
 
 Monitor individual commit histories, track who is actively engaged, identify alumni or new contributors, and even compare the performance of contributors against each other. This encompasses a wide range of metrics such as code commits, created and reviewed Pull Requests (PRs), as well as issue creation and commenting.
 
+Contributors can use Lists feature to track their own activity, while maintainers can use the feature to manage the community.
+
 ![lists-page](../../static/img/lists-page.svg)
 
-## How to Create a List
+## Why Create Lists for Contributors and Maintainers?
```

Comment:
Let's cut the heading here and jump into two headings:

## Lists for Maintainers

Lists offer a variety of opportunities to track contributors for various reasons, including:

## Lists for Contributors

Lists offer contributors a way to gain insight into the world of open source contributions, including: